central/graphql/resolvers/image_cve_core_test.go

```diff
@@ -127,3 +130,39 @@ func (s *ImageCVECoreResolverTestSuite) TestCountImageCVEsWithQuery() {
 	s.NoError(err)
 	s.Equal(response, int32(3))
 }
+
+func (s *ImageCVECoreResolverTestSuite) TestGetImageCVEMalformed() {
+	_, err := s.resolver.ImageCVE(s.ctx, struct{ Cve *string }{})
+	s.Error(err)
+}
+
+func (s *ImageCVECoreResolverTestSuite) TestGetImageCVENonEmpty() {
+	expectedQ := search.NewQueryBuilder().AddExactMatches(search.CVE, "cve-xyz").ProtoQuery()
+	expected := []imagecve.CveCore{
+		imageCVEViewMock.NewMockCveCore(s.mockCtrl),
+	}
+
+	s.imageCVEView.EXPECT().Get(s.ctx, expectedQ, views.ReadOptions{}).Return(expected, nil)
+	response, err := s.resolver.ImageCVE(s.ctx, struct{ Cve *string }{Cve: pointers.String("cve-xyz")})
+	s.NoError(err)
+	s.NotNil(response.data)
+}
+
+func (s *ImageCVECoreResolverTestSuite) TestGetImageCVENonEmptyUnexpected() {
+	expectedQ := search.NewQueryBuilder().AddExactMatches(search.CVE, "cve-xyz").ProtoQuery()
+	expected := []imagecve.CveCore{
```

Comment:
Looks like this test case causes a panic, which is expected. However, Go Unit tests job on CI fails because of the panic.